server/graphql/v2/query/collection/ContributorsCollectionQuery.ts

```diff
@@ -0,0 +1,153 @@
+import assert from 'assert';
+
+import { GraphQLBoolean, GraphQLInt, GraphQLList, GraphQLNonNull } from 'graphql';
+import { isNil } from 'lodash';
+
+import { ifStr } from '../../../../lib/utils';
+import { sequelize } from '../../../../models';
+import { BadRequest } from '../../../errors';
+import { GraphQLContributorCollection } from '../../collection/ContributorCollection';
+import { AccountTypeToModelMapping, GraphQLAccountType } from '../../enum/AccountType';
+import { GraphQLMemberRole } from '../../enum/MemberRole';
+import { fetchAccountWithReference, GraphQLAccountReferenceInput } from '../../input/AccountReferenceInput';
+import { GraphQLChronologicalOrderInput } from '../../input/ChronologicalOrderInput';
+import EmailAddress from '../../scalar/EmailAddress';
+
+const ContributorsCollectionQuery = {
+  description: 'Get Contributors grouped by their profiles',
+  type: new GraphQLNonNull(GraphQLContributorCollection),
+  args: {
+    account: {
+      type: GraphQLAccountReferenceInput,
+      description: 'Host hosting the account',
+    },
+    host: {
+      type: GraphQLAccountReferenceInput,
+      description: 'Host hosting the account',
+    },
+    limit: { type: new GraphQLNonNull(GraphQLInt), defaultValue: 100 },
+    offset: { type: new GraphQLNonNull(GraphQLInt), defaultValue: 0 },
+    role: { type: new GraphQLList(GraphQLMemberRole) },
+    type: { type: new GraphQLList(GraphQLAccountType) },
+    email: {
+      type: EmailAddress,
+      description: 'Admin only. To filter on the email address of a member, useful to check if a member exists.',
+    },
+    orderBy: {
+      type: new GraphQLNonNull(GraphQLChronologicalOrderInput),
+      defaultValue: { field: 'createdAt', direction: 'ASC' },
+      description: 'Order of the results',
+    },
+    includeInherited: {
+      type: GraphQLBoolean,
+      defaultValue: true,
+    },
+  },
+  async resolve(_: void, args, req: Express.Request) {
+    // Check Pagination arguments
+    if (isNil(args.limit) || args.limit < 0) {
+      args.limit = 100;
+    }
+    if (isNil(args.offset) || args.offset < 0) {
+      args.offset = 0;
+    }
+    if (args.limit > 100 && !req.remoteUser?.isRoot()) {
+      throw new Error('Cannot fetch more than 1,000 members at the same time, please adjust the limit');
+    }
+
+    assert(args.account || args.host, 'You must provide either an account or a host to fetch the contributors');
```

Comment:
Can't use both at the same time (xor)

```suggestion
    assert(Boolean(args.account) ^ Boolean(args.host), 'You must provide either an account or a host to fetch the contributors');
```